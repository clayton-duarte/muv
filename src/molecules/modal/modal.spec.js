import React from 'react';

import Modal from '.';

it('should render a modal', () => {
  const wrapper = mount(
    // MOUNT METHOD CREATES SNAPSHOTS TOO MUCH DETAILED, USE CAREFULLY
    <Modal />,
  );
  expect(wrapper).toMatchSnapshot();
});
