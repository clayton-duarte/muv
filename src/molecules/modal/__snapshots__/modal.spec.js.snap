// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a modal 1`] = `
<Modal
  noButton={false}
  visible={false}
>
  <Button
    disabled={false}
    href={null}
    onClick={[Function]}
    outline={false}
    secondary={false}
  >
    <styled.button
      disabled={false}
      href={null}
      onClick={[Function]}
      outline={false}
      secondary={false}
      theme={
        Object {
          "color": Object {
            "black": "#222222",
            "darkgray": "#515151",
            "error": "#dd3333",
            "gray": "#a9abae",
            "lightgray": "#e5e5e5",
            "overlay": "rgba(0, 0, 0, .15)",
            "primary": "#0081c8",
            "secondary": "#00245f",
            "success": "#33bb33",
            "warn": "#dd3333",
            "white": "#fefefe",
          },
          "font": Object {
            "color": "#222222",
            "family": "system-ui, sans-serif",
            "size": "16px",
            "spacing": 0,
          },
          "shape": Object {
            "border": ".125rem solid rgba(0, 0, 0, .15)",
            "margin": ".25rem .125rem",
            "padding": ".75rem .875rem",
            "radius": ".25rem",
            "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
            "size": "16px",
          },
          "transition": Object {
            "time": ".3s ease",
          },
        }
      }
    >
      <button
        className="sc-bwzfXH gxSygZ"
        disabled={false}
        href={null}
        onClick={[Function]}
      >
        test
      </button>
    </styled.button>
  </Button>
  <Overlay
    onClick={[Function]}
    visible={false}
  >
    <styled.aside
      onClick={[Function]}
      theme={
        Object {
          "color": Object {
            "black": "#222222",
            "darkgray": "#515151",
            "error": "#dd3333",
            "gray": "#a9abae",
            "lightgray": "#e5e5e5",
            "overlay": "rgba(0, 0, 0, .15)",
            "primary": "#0081c8",
            "secondary": "#00245f",
            "success": "#33bb33",
            "warn": "#dd3333",
            "white": "#fefefe",
          },
          "font": Object {
            "color": "#222222",
            "family": "system-ui, sans-serif",
            "size": "16px",
            "spacing": 0,
          },
          "shape": Object {
            "border": ".125rem solid rgba(0, 0, 0, .15)",
            "margin": ".25rem .125rem",
            "padding": ".75rem .875rem",
            "radius": ".25rem",
            "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
            "size": "16px",
          },
          "transition": Object {
            "time": ".3s ease",
          },
        }
      }
      visible={false}
    >
      <aside
        className="sc-bdVaJa ihgAJq"
        onClick={[Function]}
      >
        <styled.div
          noButton={false}
          onClick={[Function]}
          theme={
            Object {
              "color": Object {
                "black": "#222222",
                "darkgray": "#515151",
                "error": "#dd3333",
                "gray": "#a9abae",
                "lightgray": "#e5e5e5",
                "overlay": "rgba(0, 0, 0, .15)",
                "primary": "#0081c8",
                "secondary": "#00245f",
                "success": "#33bb33",
                "warn": "#dd3333",
                "white": "#fefefe",
              },
              "font": Object {
                "color": "#222222",
                "family": "system-ui, sans-serif",
                "size": "16px",
                "spacing": 0,
              },
              "shape": Object {
                "border": ".125rem solid rgba(0, 0, 0, .15)",
                "margin": ".25rem .125rem",
                "padding": ".75rem .875rem",
                "radius": ".25rem",
                "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
                "size": "16px",
              },
              "transition": Object {
                "time": ".3s ease",
              },
            }
          }
          visible={false}
        >
          <div
            className="sc-bxivhb lefmte"
            onClick={[Function]}
          >
            content
          </div>
        </styled.div>
      </aside>
    </styled.aside>
  </Overlay>
</Modal>
`;
